include/roboteam_utils/networking/CMakeLists.txt

```
@@ -0,0 +1,24 @@
+#include(ccache)
+#include(cotire)
+
+# find the protobuf compiler and libraries
+find_package(zeroMQ REQUIRED)
+find_package(zmqpp 4.1.2 REQUIRED)
+
+add_library(networking STATIC
+        src/Channel.cpp
+        src/Channels.cpp
+        )
+
+target_include_directories(networking
+        PUBLIC ./include
```

./? can't you just use "include" ?

---

Faily confident i just copied it.
```suggestion
        PUBLIC include
```